### DISCUSSION TITLE
How to serialize Dashboard ?

### DISCUSSION THREAD

**Discussion by CharlotteAndre** [jbaysolutions/vue-grid-layout#628] [Q&A]
Hi, i'm pretty new to JS Framework and vue-grid-layout and as I was reading documentation I didn't find anything about a potentiel serialization methods ( i know it's possible !). I've created a dashboard made of different component and I would like my users to be able to save and restore their works but there no explanation on how to do it. Any advices ?

**Comment by jvjvjv** [comment]
You could save the grid's `layout` prop itself. You can serialize the object in your code with the statement, `JSON.stringify(this.layout)`. Save that when a user wants to save their work, and then when they load, run `this.layout = JSON.parse(savedLayout)`.
